framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -863,29 +869,11 @@ public AnnotationMirror getAnnotationMirror(Tree tree, Class<? extends Annotatio
     }
 
     /**
-     * Determines the annotated type of an AST node.
-     *
-     * <p>
-     *
-     * The type is determined as follows:
-     * <ul>
-     *  <li>if {@code tree} is a class declaration, determine its type via
-     *    {@link #fromClass}</li>
-     *  <li>if {@code tree} is a method or variable declaration, determine its
-     *    type via {@link #fromMember(Tree)}</li>
-     *  <li>if {@code tree} is an {@link ExpressionTree}, determine its type
-     *    via {@link #fromExpression(ExpressionTree)}</li>
-     *  <li>otherwise, throw an {@link UnsupportedOperationException}</li>
-     * </ul>
+     * Returns an AnnotatedTypeMirror representing the qualified type of {@code tree}.
      *
      * @param tree the AST node
-     * @return the annotated type of {@code tree}
-     *
-     * @see #fromClass(ClassTree)
-     * @see #fromMember(Tree)
-     * @see #fromExpression(ExpressionTree)
+     * @return the qualified type of {@code tree}
```

Comment:
Here it's `qualified type` and on line 965 you changed `annotated type` to `AnnotatedTypeMirror`.
I like annotated type better; qualified type sounds too similar to fully-qualified type.